src/main/java/fish/payara/micro/maven/MavenProject.java

```diff
@@ -79,6 +79,9 @@ public class MavenProject extends PayaraMicroProject {
     private static final String EXPLODED = "exploded";
     private static final String EXPLODED_PROPERTY = "-Dexploded=true";
     public static final String DEPLOY_WAR_PROPERTY = "-DdeployWar=true";
+    public static final String PAYARA_TRANSFORMER = "fish.payara.transformer";
+    public static final String PAYARA_TRANSFORMER_MAVEN = "fish.payara.transformer.maven";
+    public static final String PAYARA_TRANSFORMER_VERSION = "0.2.10";
```

Comment:
please move to the last release of the plugin on maven central 0.2.11

Reply:
done!